### PR TITLE
feat(metrics): add actor execution time metric

### PR DESCRIPTION
**Pull request by MingjiHan99**
## What's changed and what's your intention?

Use `total_poll_duration` in `tokio_metrics` to get the cumulative execution time of actors.

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

### CODE REVIEW

File: src/stream/src/task/stream_manager.rs [hunk 1]
```diff
@@ -707,6 +727,8 @@ impl LocalStreamManagerCore {
     fn drop_all_actors(&mut self) {
         for (actor_id, handle) in self.handles.drain() {
             self.context.retain(|&(up_id, _)| up_id != actor_id);
+            self.actor_monitor_tasks.get(&actor_id).unwrap().abort();
+            self.actor_monitor_tasks.remove(&actor_id);
```
Comment (jon-chuang) [inline src/stream/src/task/stream_manager.rs:731]: can just change the above two lines to `remove.unwrap().abort()`

**Comment by jon-chuang** [comment]
LGTM, this will be very useful, thanks!

File: src/stream/src/task/stream_manager.rs (hunk 1 above)
Comment (MingjiHan99) [reply src/stream/src/task/stream_manager.rs:731]: Sure

File: src/stream/src/task/stream_manager.rs [hunk 2]
```diff
@@ -643,13 +646,29 @@ impl LocalStreamManagerCore {
...
+            let actor_id_str = actor_id.to_string();
+
+            let metrics = self.streaming_metrics.clone();
+            let task = tokio::spawn(async move {
+                loop {
+                    metrics
+                        .actor_execution_time
+                        .with_label_values(&[&actor_id_str])
+                        .set(monitor.cumulative().total_poll_duration.as_secs_f64());
+                    tokio::time::sleep(Duration::from_secs(1)).await;
```
Comment (skyzh) [inline src/stream/src/task/stream_manager.rs:668]: Prometheus collects metrics by 15s, we can set this to a larger value like 5s.

**Review by skyzh — APPROVED** [review]
LGTM, but I'm not sure whether this metrics makes sense to you.

https://docs.rs/tokio-metrics/latest/tokio_metrics/struct.TaskMetrics.html#structfield.total_poll_duration

Seems to be the same as the total time of running the task.

**Pull request merged**